src/trait_manager/index.js

```diff
@@ -100,6 +100,7 @@ export default () => {
         config: c
       });
       TraitsViewer.itemsView = types;
+      TraitsViewer.updatedCollection();
```

Comment:
This addresses [this issue](https://github.com/artf/grapesjs/issues/3735#issuecomment-913921482)